Dockerfile

```diff
@@ -91,7 +91,7 @@ RUN pip install -r requirements-dev.txt
 RUN pip install tox==2.1.1
 
 ADD . /code/
-RUN python setup.py install
+RUN pip install --no-deps -e /code
```

Comment:
Yup, exactly. Previously this was installing new versions of dependencies .Now it doesn't do any re-install of dependencies, so only what's listed in `requirements.txt` gets installed (but if we somehow miss a dependency in requirements.txt it fails, which I think is good).